interfaces/builtin/u2f_devices.go

```diff
@@ -182,6 +182,11 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "15e1",
 		ProductIDPattern: "2019",
 	},
+	{
+		Name:             "Kensington VeriMark Guard Fingerprint Key",
+		VendorIDPattern:  "047d",
+		ProductIDPattern: "8055",
```

Comment:
I was not able to find any resources online listing product ID for this key. Are you able to locate any documents which would confirm those IDs?

Reply:
I got them from `lsusb` on Ubuntu 22.04. As the key doesn't work with Firefox over Snap.

The Key page is at https://www.kensington.com/software/verimark-setup/verimark-guard-setup-guide/

Reply:
Sure, will do it once I get home later today.

Reply:
Here